### PR TITLE
Remove frequency_offset from received_ping.

### PR DESCRIPTION
**Pull request by kronenpj**
Per your earlier comment regarding removing frequency_offset until it's decided to fully remove or re-implement.

### CODE REVIEW

**Comment by DJ2LS** [comment]
Thanks for the cleanup! I suggest removing the entire Frequency Offset part!
Instead of changing the frequency we should consider doing this by an audio pitch and not by changing the VFO frequency of the radio. But this is not needed anymore as its meanwhile possible adjusting the freedv tuning range which solves most problems caused by an frequency Offset > +/- 50Hz

**Comment by kronenpj** [comment]
Per your request. :)

File: tnc/data_handler.py [hunk 1]
```diff
@@ -1503,7 +1486,7 @@ def received_ping(self, data_in:bytes):
         ping_frame[1:3] = static.DXCALLSIGN_CRC
         ping_frame[3:5] = static.MYCALLSIGN_CRC
         ping_frame[5:11] = static.MYGRID
-        ping_frame[11:13] = bytes([0,0]) # frequency_offset.to_bytes(2, byteorder='big', signed=True)
+        ping_frame[11:13] = bytes([0,0])
```
Comment (DJ2LS) [inline tnc/data_handler.py:1489]: This line can be removed as we don't need this information anymore.

**Comment by kronenpj** [comment]
I hadn't bothered to look up whether `bytearray` fills with zero. It does, so this can safely be removed.

**Review by DJ2LS — APPROVED** [review]

**Pull request merged**